qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -294,3 +303,84 @@ def calculate_2q_epg(
             out[qubit_pair] = {gate_2_qubit_type: 3 / 4 * (1 - alpha_c_2q) / n_gate_2q}
 
         return out
+
+
+def lookup_epg_ratio(gate: str, n_qubits: int) -> Union[None, int]:
```

Comment:
You might as well make this a static method of `RBUtils` so you dont need to import two things

Reply:
Yes I think this doesn't need to be a class. We should convert these methods into functions, but this can be done in follow-up (if you want I'll do this now)